jquery.accTabs.js

```diff
@@ -63,7 +63,7 @@
                 var tabId = self.options.tabControlId + counter + index,
                     panelId = self.options.tabPanelId + counter + index,
                     heading = $($(value).prev()),
-                    headingText = heading.text(),
+                    headingText = heading.html(),
```

Comment:
Should this variable be renamed from 'headingText' to something more appropriate such as 'headingContent'?